Makefile

```diff
@@ -250,12 +257,12 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	$(OPM) index add --container-tool $(CTR_CMD) --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
 
 # Push the catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
-	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+	$(CTR_CMD) push $(CATALOG_IMG)
 
```

Comment:
I think we should remove this as these targets are unused and  we should be using a File Based Catalog instead of sqlite based one.

Reply:
Sure I will raise a separate issue to improve the way we build the catalog.

Reply:
That is quite some work which I am not sure if we should undertake now but do go ahead with raising the issue.